src/Kafka/CreateKafkaInstance/components/TrialKafkaForm.tsx

```diff
@@ -0,0 +1,198 @@
+          validated={regionValidation}
+        />
+        {isTesting && (
+          <>
+            <Text component={TextVariants.p} className="pf-c-form__helper-text">
+              {t("cloud_region_description")}
+            </Text>
+            <Button variant={ButtonVariant.link} isInline>
+              {t("learn_about_cloud_regions")}
+            </Button>
+          </>
+        )}
+      </FormGroup>
+      <FormGroup
+        label={t("availability_zones")}
+        fieldId="availability-zones"
+        validated={azValidation}
+        helperTextInvalid={t("common:required")}
+      >
+        <ToggleGroup aria-label={t("availability_zone_selection")}>
+          <Tooltip
+            content={t("availability_zones_tooltip_message", {
+              enabledZone: azOptions?.multi ? "multi" : "single",
+            })}
+            trigger={disableAZTooltip ? "manual" : undefined}
+          >
+            <ToggleGroupItem
+              text={t("single")}
+              value={"single"}
+              isDisabled={disableControls || azOptions?.single !== true}
+              buttonId="single"
+              isSelected={az === "single"}
+              onChange={() => setAZ("single")}
+            />
+          </Tooltip>
+
+          <Tooltip
+            trigger={disableAZTooltip ? "manual" : undefined}
+            content={t("availability_zones_tooltip_message", {
+              enabledZone: azOptions?.multi ? "multi" : "single",
+            })}
+          >
+            <ToggleGroupItem
+              text={t("multi")}
+              value="multi"
+              buttonId="multi"
+              isDisabled={disableControls || azOptions?.multi !== true}
+              isSelected={az === "multi"}
+              onChange={() => setAZ("multi")}
+            />
+          </Tooltip>
+        </ToggleGroup>
+      </FormGroup>
+      {isTesting && (
+        <FormGroupWithPopover
+          labelHead={t("size")}
+          fieldId="streaming-size"
+          fieldLabel={t("size")}
+          labelBody={t("size_help_content")}
+          buttonAriaLabel={t("size_field_aria")}
+        >
+          <div className="pf-c-input-group pf-u-w-50">
+            <Slider
+              min={1}
+              max={2}
+              value={isDisabledSize ? -1 : size}
+              showTicks={true}
+              label={t("streaming_unit")}
+              className="pf-u-w-100"
+              isDisabled={isDisabledSize}
+              onChange={setSize}
+            />
+            <span
+              className="pf-c-input-group__text pf-m-plain pf-u-text-nowrap"
+              id="plain-example"
+            >
+              {t("streaming_unit")}
+            </span>
+          </div>
+          <Text component={TextVariants.p} className="pf-c-form__helper-text">
+            {t("trial_kafka_size_description")}
+          </Text>
+          <Button variant={ButtonVariant.link} isInline>
+            {t("learn_about_sizes")}
+          </Button>
```

Comment:
Same comment as above...
Ideally, the font size for this link would be the same size as the helper text. The only way to do that is by wrapping this button with the `className="pf-c-form__helper-text"` like you have above.